### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by ceztko** [hanickadot/compile-time-regular-expressions#276]

**Comment by ceztko** [comment]
P.S.: I'm sorry for initial blank issue.

**Comment by ceztko** [comment]
Thank you for the clarification. Also, can you project be used at runtime with dynamic pattern? Or not really.

**Comment by hanickadot** [comment]
Not really. The pattern is compiled-in. For best performance and safety I would recommend RE2.
